tests/test_utils.py

```diff
@@ -566,7 +566,7 @@ def generate_sample_fov_tiling_entry(coord, name):
 
     sample_fov_tiling_entry = {
         "scanCount": 1,
-        "centerPointMicrons": {
+        "centerPoinalpineercrons": {
```

Comment:
Hah, this is a pretty good control-f error

Reply:
oh wow.